editor/components/block-list/block.js

```diff
@@ -56,7 +56,7 @@ import {
 
 const { BACKSPACE, ESCAPE, DELETE, ENTER, UP, RIGHT, DOWN, LEFT } = keycodes;
 
-class VisualEditorBlock extends Component {
```

Comment:
Making this reusable means we need to use a prop or something to get a selector for the parent scrollable. https://github.com/WordPress/gutenberg/pull/3546/files#diff-bc66d5701a2fea1111bb8f1bf0150910R96

It's not guaranteed to be the same in all layouts.

Reply:
In d8be96b I reimplemented the behavior to a generic "find closest scrollable container".